packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -320,11 +325,18 @@ contract Rewards is
         require(lastProposalId >= t.nextProposalIdToReward, 'bad lastProposalId');
         require(isSortedAndNoDuplicates(votingClientIds), 'must be sorted & unique');
 
-        NounsDAOTypes.ProposalForRewards[] memory proposals = nounsDAO.proposalDataForRewards(
-            t.nextProposalIdToReward,
-            lastProposalId,
-            votingClientIds
-        );
+        uint16 proposalEligibilityQuorumBps_ = $.params.proposalEligibilityQuorumBps;
```

Comment:
nit: do we still need to cache this variable? I think we're only using it once now?

Reply:
fixed 1ebd2c4666716613632fc1a104ebfbbbbafdbe05